src/Microsoft.AspNetCore.HttpOverrides/project.json

```diff
@@ -8,7 +8,7 @@
     ],
     "xmlDoc": true
   },
-  "description": "ASP.NET Core basic middleware for:\r\nX-Forwarded-* headers to forward headers from a proxy.\r\nHTTP method override header.",
+  "description": "ASP.NET Core basic middleware for supporting HTTP method overrides. Includes:\r\nX-Forwarded-* headers to forward headers from a proxy.\r\nHTTP method override header.",
```

Comment:
Oh looks like we used "bullet points" on the other one below. So let's maybe do that here as well and put `* ` at the start of the second two lines.